drivers/spi/spi_ll_stm32.c

```diff
@@ -193,7 +193,12 @@ static void spi_stm32_complete(struct spi_stm32_data *data, SPI_TypeDef *spi,
 
 	if (LL_SPI_GetMode(spi) == LL_SPI_MODE_MASTER) {
 		while (LL_SPI_IsActiveFlag_BSY(spi)) {
-			/* NOP */
+			/* BSY flag is set when MODF is raised (but not only).
```

Comment:
BSY but is cleared when MODF is raised.
So MODF clearing should be done out of while loop

Reply:
MODF clearance has been placed out of the loop, thank you !